samples/storage_csm_powermax_v2130.yaml

```diff
@@ -40,7 +40,6 @@ spec:
     # Default value: 2
     replicas: 2
     # Default credential secret for Powermax, if not set it to ""
-    authSecret: powermax-creds
```

Comment:
If this version of Operator does not handle auth secrets for PowerMax, should we remove such entries in older powermax sample files?

Reply:
I mean we could .. but we wont be using those sample files in future

Reply:
Operator supports up to n-2 version of driver installs, right. So, user may use those.

Reply:
oh in this case then I can make a change in .12 and .11 as well.

Reply:
Done !